<launch>
    <node name="manip_interface" pkg="manip_control" type="manip_gazebo_interface.py" output="screen">
		<rosparam file="$(find manip_control)/config/manip_gazebo_interface.yaml" command="load"/>
    </node>
</launch>